### PR TITLE
feat: cult inquisitor, hider and seeker

### PR DESCRIPTION
**Pull request by Golbolco**
renamed mafia's "inquisitor" role to Hider based on a suggestion from shwartz99

built a new Inquisitor for the Cult faction based on https://wiki.mafiascum.net/index.php?title=Inquisitor

intended effect: nightkiller. if their target is protected during the night, the target will not die and instead convert to cultist

### CODE REVIEW

**Comment by Golbolco** [comment]
was recommended to me that a Cult equivalent would be good too. called him "it"

if he visits either the seeker or hider, they convert into cultists

**Comment by Golbolco** [comment]
renamed for another lovecraft reference

File: Games/types/Mafia/roles/Cult/Kamog.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    super("Kamog", player, data);
+
+    this.alignment = "Cult";
+    this.cards = [
+      "VillageCore",
+      "WinWithCult",
+      "MeetingCult",
+      "GuessAdversaryConvert",
+    ];
+    this.roleToGuess = "Seeker" && "Hider";
```
Comment (pikulet) [inline Games/types/Mafia/roles/Cult/Kamog.js:14]: broken

File: Games/types/Mafia/roles/cards/Vigicultist.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      "Kill Unbelievers": {
+        actionName: "Kill",
+        states: ["Night"],
+        flags: ["voting"],
+        action: {
+          labels: ["kill"],
+          priority: PRIORITY_KILL_DEFAULT,
+          run: function () {
+            if (this.dominates()) this.target.kill("basic", this.actor);
+            else if (action.hasLabel("save")) {
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/Vigicultist.js:18]: broken

**Comment by pikulet** [comment]
seeker and inquisitor aren't used much now, would this add value to the site's role list?

**Comment by Golbolco** [comment]
> seeker and inquisitor aren't used much now, would this add value to the site's role list?

We've had setups use them before to great effect, art assets have been prepped for them, and building on the roleToGuess function only opens up possibilities for future devs, so I think it adds a lot of value.

**Pull request merged**